lib/serialization_helper.rb

```diff
@@ -122,28 +122,52 @@ def self.unhash_records(records, keys)
     end
 
     def self.convert_booleans(records, columns)
-      records.each do |record|
+      records.map do |record|
```

Comment:
@hzeus Interestingly, this change _is needed_ with Rails 5, because `select_all` now returns an `ActiveRecord::Result`, which in turn [yields a copy of each row](https://github.com/rails/rails/blob/d9fd1479181b8be6c080e9b3186fb48283ba68df/activerecord/lib/active_record/result.rb#L52-L59). So it can't be modified in-place.

It would also be possible to use `to_ary` (which returns the underlying array) instead, but I find this approach to be cleaner.